core/transact.go

```diff
@@ -223,15 +222,15 @@ func (h *Handler) finalizeTxWait(ctx context.Context, txTemplate *txbuilder.Temp
 	return nil
 }
 
-func waitForTxInBlock(ctx context.Context, c *protocol.Chain, tx *bc.Tx, height uint64) (uint64, error) {
+func (h *Handler) waitForTxInBlock(ctx context.Context, tx *bc.Tx, height uint64) (uint64, error) {
```

Comment:
Oh right. I guess I was thinking we could convert this to a method first, and then this PR would only touch the FinalizeTx call in this method. But not a big deal.